src/lib/modifiers.ts

```diff
@@ -0,0 +1,14 @@
+import { $familiar } from "libram";
+
+import { runsOrNCFamiliar } from "./familiar";
+
+export function getModString(): string {
+  if (
+    runsOrNCFamiliar() === $familiar`Frumious Bandersnatch` ||
+    runsOrNCFamiliar() === $familiar`Pair of Stomping Boots`
+  ) {
+    return "-combat, 0.25 familiar weight"; // This way 1 free run counts for slightly more than a softcapped combat-rate modifier
```

Comment:
Can we restructure this method a little, to simplify the return statement?

What I have in mind is something like
```
let fwWeight = "0.02"
if (banderBoots) {
    fwWeight = "0.25"
}
return `-combat, ${fwWeight} familiar weight`;
```